src/videodownlinkbandwidthpolicy/VideoAdaptiveProbePolicy.ts

```diff
@@ -3,15 +3,30 @@
 
 import ContentShareConstants from '../contentsharecontroller/ContentShareConstants';
 import Logger from '../logger/Logger';
+import ServerSideNetworkAdaption from '../signalingclient/ServerSideNetworkAdaption';
 import VideoStreamDescription from '../videostreamindex/VideoStreamDescription';
 import VideoStreamIndex from '../videostreamindex/VideoStreamIndex';
 import VideoPreference from './VideoPreference';
 import { VideoPreferences } from './VideoPreferences';
 import VideoPriorityBasedPolicy from './VideoPriorityBasedPolicy';
+import VideoPriorityBasedPolicyConfig from './VideoPriorityBasedPolicyConfig';
 
+/** [[VideoAdaptiveProbePolicy]] wraps [[VideoPriorityBasedPolicy]] with customized behavior to automatically
+ * assign a high preference to content share.
+ *
+ * @deprecated This class is not compatible with latest priority policy features (i.e. server side network adaptation),
```

Comment:
1. Since we are trying to help the builder with the documenation, probably a link to follow when we say "can be done trivially by the application" would be helpful.
2. This seems like a good detail for changelog entry as a build can understand the change better.

Reply:
Will update!

Reply:
actually i misunderstood a comment i wrote before/how this is used (when simulcast is enabled but a policy is not set). I don't need to deprecate this yet.